google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -161,7 +161,7 @@ class Line:
             Required. UTF-8 encoded text.
     """
 
-    documentai_line: documentai.Document.Page.Line
+    documentai_line: dataclasses.InitVar[documentai.Document.Page.Line]
     text: str
```

Comment:
It feels like we only need the `InitVar` `documentai_line` here, and we can extract the text and store that in a `Line.text` field in `__post_init__` (which has access to `documentai_line`)?

That is, perhaps the `text` field should be a dataclass field with `init=False`, so the caller does not have to pass in both `documentai_line` and `text`?

(Similar situation in some other classes that currently do not have `__post_init__`.)

Reply:
I agree